scipy/stats/_multivariate.py

```diff
@@ -5057,6 +5059,168 @@ def rvs(self, size=1, random_state=None):
                               size=size,
                               random_state=random_state)
 
+class dirichlet_multinomial_gen(multi_rv_generic):
+    r"""A Dirichlet multinomial random variable.
```

Comment:
I don't mean to be too nitpicky and this can be done in a follow-up but would you be able to present a usage example to add to the docstring? Even better if it could show when this would be more appropriate than other distributions.

Reply:
I can add an example of how to use the functions if you want, but I don't think explaining when to use it is a good idea. If somebody is looking to use the Dirichlet multinomial distribution, then it's safe to assume that they know when to use it.

Reply:
Please include notes and examples like other multivariate distributions.